website/data/intro-navigation.js

```diff
@@ -8,21 +8,14 @@
 
 export default [
   'index',
-  { category: 'vs', content: ['cli-tools', 'docker', 'terraform'] },
   {
-    category: 'getting-started',
+    category: 'vs',
     content: [
-      'install',
-      'project_setup',
-      'boxes',
-      'up',
-      'synced_folders',
-      'provisioning',
-      'networking',
-      'share',
-      'teardown',
-      'rebuild',
-      'providers',
+      { title: 'Getting Started', href: 'https://learn.hashicorp.com/vagrant' },
```

Comment:
Hmm, I'm not sure if this is the navigation file that's doing this...or if it's something that hasn't been changed yet, but when I'm on the `intro` page I see a `Getting Started` link in the nav bar. However it takes me to `http://localhost:3000/intro/getting-started` which ends up being a 404, rather than directing to the learning site :thinking:

Reply:
Oh weird. Have you tried it on the [Netlify build](https://deploy-preview-11774--vagrant-docs.netlify.app/intro)? The website's README file says that redirects don't work correctly on the local build, but I think it's ok.

Reply:
Yeah! Ok the Netlify build works :) I didn't realize the redirects won't work locally.